### PR TITLE
Etl/download india districts data

### PR DESCRIPTION
**Pull request by ceilingloft**
Added download_data function to india districts etl file (Issue #23 )

### CODE REVIEW

File: studies/india_districts/etl.py [hunk 1]
```diff
@@ -168,6 +172,24 @@
...
+current_version = 4
+
+def download_data(data_path: Path):
+    base_url = 'https://api.covid19india.org/csv/latest/'
+    today = datetime.today().strftime('%Y-%m-%d')
+
+    for i in range(current_version):
+        api_file = 'raw_data' + str(i + 1) + '.csv'
+        filename = today + '_' + api_file
+        if not os.path.exists(data_path/filename):
```
Comment (satejsoman) [inline studies/india_districts/etl.py:184]: `Path` objects also have `.exists()` and `.open()` methods - prefer consistency with `glob` below

File: studies/india_districts/etl.py [hunk 2]
```diff
@@ -168,6 +172,24 @@
     'Num cases'
 ]
 
+current_version = 4
```
[the remark this answers is not in the document]
Comment (ceilingloft) [reply studies/india_districts/etl.py:175]: are you suggesting just a change of the variable name or that I change the way the code works with this? please can you let me know briefly how you would suggest I change it?

File: studies/india_districts/etl.py [hunk 3]
```diff
@@ -168,6 +172,24 @@
...
+    today = datetime.today().strftime('%Y-%m-%d')
+
+    for i in range(current_version):
+        api_file = 'raw_data' + str(i + 1) + '.csv'
+        filename = today + '_' + api_file
+        if not os.path.exists(data_path/filename):
+            url = base_url + api_file
+            response = requests.get(url)
+
+            for current_file in data_path.glob('*' + api_file):
```
[the remark this answers is not in the document]
Comment (ceilingloft) [reply studies/india_districts/etl.py:188]: ok, I thought this probably wasn't the best idea but I wasn't sure how to overwrite the files as I didn't want to assume what the previous files would be called

File: studies/india_districts/lockdown_evaluation.py [hunk 4]
```diff
@@ -1,6 +1,7 @@
 from itertools import product
 from pathlib import Path
 from typing import Dict, Optional, Sequence
+from datetime import datetime
```
Comment (satejsoman) [inline studies/india_districts/lockdown_evaluation.py:4]: don't think we need this import any more?

File: studies/india_districts/requirements.txt [hunk 5]
```diff
@@ -0,0 +1,56 @@
+-e git+https://github.com/mansueto-institute/adaptive-lockdown@574e31d27273f90b5c9a01bf11f9292f4cc24b25#egg=adaptive
```
Comment (satejsoman) [inline studies/india_districts/requirements.txt:1]: move the output of pip freeze to the repo root

Comment (ceilingloft) [reply studies/india_districts/requirements.txt]: sorry I thought I had done this

File: studies/india_districts/lockdown_evaluation.py (hunk 4 above)
Comment (ceilingloft) [reply studies/india_districts/lockdown_evaluation.py]: deleted

**Pull request merged**